icechunk/src/repository.rs

```diff
@@ -455,6 +455,10 @@ impl Repository {
         &self.asset_manager
     }
 
+    pub fn authorized_virtual_containers(&self) -> &HashMap<String, Option<Credentials>> {
```

Comment:
To decrease surface area for now, do yuou want to just make this match the pyuthon impl and just return prefixes for now?